Makefile

```diff
@@ -604,7 +604,16 @@ $(protoc_gen_go_bin): | go-check
 	@echo "Installing protoc-gen-go $(protoc_gen_go_version)..."
 	$(E)rm -rf $(protoc_gen_go_base_dir)
 	$(E)mkdir -p $(protoc_gen_go_dir)
-	$(E)$(go_path) go build -o $(protoc_gen_go_bin) github.com/golang/protobuf/protoc-gen-go
+	$(E)$(go_path) go build -o $(protoc_gen_go_bin) google.golang.org/protobuf/cmd/protoc-gen-go
+
+install-protoc-gen-go-grpc: $(protoc_gen_go_grpc_bin)
+
+$(protoc_gen_go_grpc_bin): | go-check
+	@echo "Installing protoc-gen-go-grpc $(protoc_gen_go_grpc_version)..."
+	$(E)rm -rf $(protoc_gen_go_grpc_base_dir)
+	$(E)mkdir -p $(protoc_gen_go_grpc_dir)
+	$(E)echo "module tools" > $(protoc_gen_go_grpc_dir)/go.mod
+	$(E)cd $(protoc_gen_go_grpc_dir) && GOBIN=$(protoc_gen_go_grpc_dir) $(go) get google.golang.org/grpc/cmd/protoc-gen-go-grpc@$(protoc_gen_go_grpc_version)
```

Comment:
I think this line didn't get merged properly. It looks like `$(go)` should be replaced with `$(E)$(go_path) go`.